SoSe19/bericht_data.tex

```diff
@@ -0,0 +1,80 @@
+Das Ziel dieser Fortgeschrittenenpraktika ist die Vermittlung von spezifischen inhaltlichen sowie formellen Qualifikationszielen und Schlüsselqualifikationen. Diese Ziele sind zentrale Fähigkeiten Studierender der Physik und sollen dabei im Fortgeschrittenenpraktikum erlernt und ausgebaut werden. Nach erfolgreichem Abschluss sollen Qualifikationen wie vertiefte Statistik- und Plotkenntnisse, ein nachhaltiges Forschungsdatenmanagement sowie ein übersichtliches Dokumentieren und Auswerten der gemessenen Daten vermittelt worden sein. Übergeordnetes Ziel ist dabei, eine Intuition für physikalische Zusammenhänge zu entwickeln. Die Gestaltung und Vermittlung dieser Qualifikationsziele obliegt dabei der Hochschule. 
+
+\section*{Programmierkentnisse}
+
+Die Vermittlung von Programmierkenntnissen ist ein vitaler Teil des Physikstudiums. Seit einiger Zeit beschäftigt sich die ZaPF daher damit, wie diese idealerweise im Studium integriert sein sollte. Auf der letzten ZaPF  In Bonn wurde deswegen hierzu ein Leitfaden \footnote{\url{https://zapf.wiki/Leitfaden_Programmierkentnisse_im_Physikstudium}} erstellt.
+Hierbei werden Präsenzprogrammierübungen im Gegensatz zu reinem Frontalunterricht als geeignete Methode zur Vermittlung von solchen Fähigkeiten vorgeschlagen.
+
+\section*{Wissenschaftskommunikation}
+
+Seit dem Wintersemester 2016 beschäftigt sich die ZaPF mit dem Thema  Wissenschaftskommunikation (siehe Postionspapiere von 2018 \footnote{\url{https://zapfev.de/resolutionen/wise18/PosPap_WissKomm_I/WissKomm_I.pdf}} \footnote{\url{https://zapfev.de/resolutionen/wise18/PosPap_WissKomm_II/WissKomm_II.pdf}}. In diesem Semester wurden die erarbeiteten Forderungen und Umsetzungshinweise  zusammengefasst, die auf eigenen Erfahrungen, Gesprächen mit Wissenschaftskommunikator*innen und Schulungen basieren. Außerdem wurde sich ausgetauscht, wie man sich in Diskussionen zu Pseudowissenschaften und falschen Darstellungen am besten äußert.
+
+\section*{Nachhaltigkeit und Diversität}
+
+Ein weiteres Thema war die Nachhaltigkeit an Hochschulen, in der Fachschaftsarbeit und dem eigenen Handeln. Aus dem Austausch entstand eine Liste mit Handlungsvorschlägen. Außerdem werden Arbeitskreise auf den kommenden ZaPFen folgen, um ein nachhaltiges Handeln langfristig sowohl in der Fachschaftsarbeit als auch auf der ZaPF und im Privatleben zu etablieren. Auch über den Umgang mit Geschlechterdiversität und gendergerechte Sprache wurde beraten.
+
+\section*{Allgemeine Hochschulpolitik}
+
+Neben Themen mit direktem Bezug zum Physikstudium hat die ZaPF sich auch Themen der allgemeineren Hochschulpolitik gewidmet. Hierzu zählen Positionierungen gegen Denunziationsplattformen für Lehrpersonen, für eine demokratische Hochschule oder eine Erneuerung der Position, dass Verfasste Studierendenschaften zur Vertretung der Rechte und Interessen der Studierenden notwendig sind.
```

Comment:
Hier wäre es auch schön, wenn die verabschiedeten Resolutionen auch verlinkt wären. Außerdem gab es noch eine Resolution zur Zivilklausel in NRW, die könnte man hier auch erwähnen.

Reply:
Das ist eine redaktionelle Änderung, die auch bei dem schon verabschiedeten Bericht problemlos möglich sein sollte.